Proposals/xgov-10.md

```diff
@@ -0,0 +1,53 @@
+---
+period: 10
+title: YourPlace
+author: Austin Lawrence (nops.algo)
+company_name: YourPlace Inc
+category: dApp
+focus_area: Identity
+open_source: Yes
+amount_requested: 1000000
+status: Draft
+---
+
+## Abstract
+[YourPlace](https://yourplace.network) is a decentralized, uncensorable, social media protocol build on IPFS and the Algorand blockchain. This open source software was built to distribute social media posts, profiles and content.
```

Comment:
```suggestion
<a href="https://yourplace.network">YourPlace</a> is a decentralized, uncensorable, social media protocol build on IPFS and the Algorand blockchain. This open source software was built to distribute social media posts, profiles and content.
```

Reply:
Is the standard for xGov requests to use HTML links, instead of Markdown?